src/components/selection_input/selection_input.xml

```diff
@@ -25,11 +29,13 @@
             t-on-keydown="onKeydown"
             t-att-value="range.xc"
             t-att-style="getColor(range)"
+            t-att-title="range.disabled ? props.disabledTitle : ''"
```

Comment:
```suggestion
            t-att-title="range.disabled ? props.disabledTitle : undefined"
```
nitpick. I think this avoids having empty title in the HTML and having to change the snapshots